Makefile

```diff
@@ -0,0 +1,9 @@
+PYTHON ?= python
+
+all: linter
+
+cpplint.py:
+	curl -o cpplint.py "https://raw.githubusercontent.com/nodejs/node/master/tools/cpplint.py"
```

Comment:
You may want to pin it to a specific release (e.g. `v7.5.0` instead of `master`) in case it gets updated to different settings.